src/sentry/web/frontend/base.py

```diff
@@ -220,25 +247,30 @@ def redirect_to_org(self, request: Request):
 
         return HttpResponseRedirect(url)
 
+    @abc.abstractmethod
+    def respond(
+        self, template: str, context: Mapping[str, Any] | None = None, status: int = 200
+    ) -> HttpResponse:
+        raise NotImplementedError
 
-class BaseView(View, OrganizationMixin):
-    auth_required = True
-    # TODO(dcramer): change sudo so it can be required only on POST
-    sudo_required = False
-
-    csrf_protect = True
 
-    def __init__(self, auth_required=None, sudo_required=None, csrf_protect=None, *args, **kwargs):
-        if auth_required is not None:
-            self.auth_required = auth_required
-        if sudo_required is not None:
-            self.sudo_required = sudo_required
-        if csrf_protect is not None:
-            self.csrf_protect = csrf_protect
+class BaseView(View, OrganizationMixin):  # type: ignore
+    def __init__(
+        self,
+        auth_required: bool = True,
+        sudo_required: bool = False,
+        csrf_protect: bool = True,
+        *args: Any,
+        **kwargs: Any,
+    ) -> None:
+        self.auth_required = auth_required
+        # TODO(dcramer): change sudo so it can be required only on POST
+        self.sudo_required = sudo_required
+        self.csrf_protect = csrf_protect
```

Comment:
Looks like you might have to revert this chunk:
https://stackoverflow.com/questions/24244245/django-passing-an-attribute-value-to-as-view-in-templateview